modules/hdf4_handler/hc2dap.cc

```diff
@@ -359,7 +401,7 @@ HDFGrid *NewGridFromSDS(const hdf_sds & sds, const string &dataset)
 // return 0 if the HDF Type is invalid or not supported.
 BaseType *NewDAPVar(const string &varname,
 		    const string &dataset,
-		    int32 hdf_type)
+		    const int32 hdf_type)
```

Comment:
Curious: Did SonarScan suggest this?

Reply:
No, this is a mistake I introduced. I will retract the change.